guide/payments/receive.md

```diff
@@ -0,0 +1,215 @@
+This page should cover what to do when receiving bitcoin, how to share and copy addresses etc.
+
+- What addresses to support (native segwit by default)
+- How to present QR codes (allow for specifying an amount)
+- Generating a new address 
+
+-->
+
+For someone to receive bitcoin, the sender needs to know the recipient’s address. The receiver generates an address in their Bitcoin application and shares it with the sender as text, by scanning a [QR Code]({{ '/guide/foundations/wallet-interoperability/#qr-codes' | relative_url }}). [Payment links]({{ '/guide/foundations/wallet-interoperability/#payment-links' | relative_url }}) are also used to share additional details about a payment and can be shared in the same way.
+
+A new address should be generated for every payment, as they should only be used once. This helps safeguards the receiver's privacy.
+
+As the blockchain only stores the amount and addresses involved in a transaction, any additional information has to be stored in your application, or encoded in payment links.
+
+It's up to you to find the balance between a payment flow that is optimized to quickly generate and share addresses, and one that prompts them to describe the payment so they can use it in the future to have more context about what payment was for.
+
+<!-- 
+Update glossary
+
+Each address that a Bitcoin application generates has an accompaniying private key. This private key allows the receiver to unlock the bitcoin that is sent to the address when the receiver chooses to spend it.
+
+Although addresses are not a problem to share publically, their accompanying private keys which typically generated from a single recovery phrase must be kept secure and private.
+-->
+
+## Preparing the payment request
+The simplest way to request bitcoin is to create and share a Bitcoin address where the sender should direct the payment. Generating a new address and showing it as a QR code and plain text is the most common way for applications to help users do so.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-compact.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-compact@2x.png"
+   alt-text = "Screen showing QR Code for the address and optional entry for other properties"
+   caption = "The receiver is able to share a bitcoin address as soon as they enter the request flow"
+   width = 250
+   height = 541
+%}
+
+</div>
+
+{% include tip/tip.html %}
+Since there are multiple types of addresses, there are some compatibility issues between them that can disrupt a payment flow. Read more about [address compatibility](#address-compatibility).
+{% include tip/close.html %}
+
+The receiver may want to request a specific amount from the sender, or save additional information for their own record. This will require extra steps and information when generating the payment request, which can then be shared with the sender as a payment link.
```

Comment:
When you say link are you referring to URI? https://en.bitcoin.it/wiki/BIP_0021. If so these can be shared with the sender as a payment link (URI) or QR.